resources/services/workspaces/workspaces.go

```diff
@@ -175,3 +182,8 @@ func resolveWorkspacesModificationStates(_ context.Context, _ schema.ClientMeta,
 	}
 	return resource.Set(c.Name, data)
 }
+func ResolveWorkspacesWorkspaceArn(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	workspace := resource.Item.(types.Workspace)
+	return resource.Set(c.Name, client.GenerateResourceARN("workspaces", "workspace", *workspace.WorkspaceId, cl.Region, cl.AccountID))
+}
```

Comment:
Isn't this trying to set the `name` attribute to the ARN value?

Could we do something like this: https://github.com/cloudquery/cq-provider-aws/blob/70e67cf0aba63326db8f605caa0a86e8c41aa85b/resources/services/apigateway/api_keys.go#L42-L43